src/components/Quotes.js

```diff
@@ -4,37 +4,58 @@ import '../quote.css';
 import ApiCredentials from './ApiCredentials';
 
 function Quotes() {
-  const [quote, setQuote] = useState('Loading...');
+  const [quote, setQuote] = useState('');
+  const [error, setError] = useState(false)
+  const [isLoading, setIsLoading] = useState(false)
   const [author, setAuthor] = useState('');
   const [category, setCategory] = useState('');
   const apiFetched = useRef(false);
 
+  // the .env will be used in production
+  // const api_url = process.env.REACT_APP_API_URL;
+
```

Comment:
- [ ] It would be great if you remove the commented variable to make your project follow the 
[JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) 👍